Makefile

```diff
@@ -73,9 +73,20 @@ generate:
 ebpf: generate
 	$(MAKE) $(EBPF_FLAGS) -C support/ebpf
 
-ebpf-profiler: generate ebpf
+ebpf-profiler: generate ebpf rust-components
 	go build $(GO_FLAGS) -tags $(GO_TAGS)
 
+rust-targets:
+	rustup target add aarch64-unknown-linux-musl
+	rustup target add x86_64-unknown-linux-musl
+
+rust-components: rust-targets
+	cargo build --lib --release --target aarch64-unknown-linux-musl
+	cargo build --lib --release --target x86_64-unknown-linux-musl
```

Comment:
Can we use TARGET_ARCH here, to avoid the requirement of installing cross-compilation tools for just building `ebpf-profiler`?